doc/source/how-to-configure-clients.rst

```diff
@@ -1,17 +1,16 @@
-Configure clients
+Configure Clients
 =================
 
-Along with model parameters, Flower can send configuration values to clients.
-Configuration values can be used for various purposes. They are, for example, a popular
-way to control client-side hyperparameters from the server.
+Flower provides the ability to send configuration values to clients, allowing
+server-side control over client behavior. This feature enables flexible and dynamic
+adjustment of client-side hyperparameters, improving collaboration and experimentation.
 
 Configuration values
 --------------------
 
 Configuration values are represented as a dictionary with ``str`` keys and values of
-type ``bool``, ``bytes``, ``double`` (64-bit precision float), ``int``, or ``str`` (or
-equivalent types in different languages). Here is an example of a configuration
-dictionary in Python:
+type ``bool``, ``bytes``, ``float``, ``int``, or ``str`` (or equivalent types in
```

Comment:
Here i'd add a reference to [Scalar](https://github.com/adap/flower/blob/8636ef555159f5592f03702b9e455bcc61c0939c/src/py/flwr/common/typing.py#L34), which is the value type supported. I don't see it being part of the docs (neither are `FitConfig`, `EvaluateConfig`) -- should these be included? @danieljanes @chongshenng